debian/snapd.install

```diff
@@ -1,4 +1,5 @@
 /usr/bin/snap
+/usr/bin/snapctl
```

Comment:
Ah, but agree regarding your point @kyrofa, that we want this in PATH for the hooks. We need to put /usr/lib/snapd in path, or move snapctl inside a subdir there and have that in PATH.